evgMrmApp/Db/evgSoftSeq.template

```diff
@@ -62,6 +62,9 @@ record(mbbo, "$(P)TrigSrc$(s=:)1-Sel") {
   field(TEST, "Univ13")
   field(ELST, "Univ14")
   field(TVST, "Univ15")
+  field(TTST, "Front2")
+  field(FTST, "Rear1")
+  field(FFST, "Rear2")
```

Comment:
Let's move it to a separate mbbo so it does not split rear0..15.

Reply:
Let's do the same so Univ0..15 == 1 mbbo - this is affected.
field(TTST, "Univ0")
field(FTST, "Univ1")
field(FFST, "Univ2")